utility_analysis_new/poisson_binomial.py

```diff
@@ -51,8 +66,16 @@ def compute_pmf_approximation(mean: float, sigma: float, skewness: float,
     The computation is based on paper chapter 3.3 (refined normal approximation)
     https://www.researchgate.net/publication/257017356_On_computing_the_distribution_function_for_the_Poisson_binomial_distribution
     """
+    if sigma == 0:
+        return PMF(int(round(mean)), np.array([1]))
     G = lambda x: (norm.cdf(x) + skewness * (1 - x * x) * norm.pdf(x) / 6)
-    xs = np.arange(-1, n + 1)
+    # n can be large. Assuming that the output distribution is close to the
+    # normal, for all i further than 8*sigma contributes less than 10^(-15).
```

Comment:
Does this affect any DP guarantees downstream? Or does this make them only tighter?
Might be helpful to add a comment.